riscv/src/register/mstatus.rs

```diff
@@ -643,11 +531,40 @@ mod test {
     #[test]
     fn test_mpp() {
```

Comment:
Missed renaming the test-case. Should probably change to `test_mstatus` for clarity, and will update with any review changes requested.